src/anemoi/inference/checkpoint.py

```diff
@@ -162,6 +163,16 @@ def target_explicit_times(self) -> Any:
         """Get the target explicit times."""
         return self._metadata.target_explicit_times
 
+    @property
+    def interpolation_window(self) -> Any:
+        """Get the interpolation window."""
+        return to_timedelta(self.data_frequency) * (self.input_explicit_times[1] - self.input_explicit_times[0])
```

Comment:
Can you put this one in the runner, it's a bit too specific to go here. The data freq can stay.